### PR TITLE
added A1_0_Compiling_Bitcoin_from_Source

### PR DESCRIPTION
**Pull request by jodobear**
Added Appendix I and public folder with screenshots from the terminal

### CODE REVIEW

**Comment by jodobear** [comment]
@shannona added the PR, waiting for your comments/approval.

File: A1_0_Compiling_Bitcoin_from_Source.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+- Basic or no familiarity with the command line and enthusiasm. The terminal is your most powerful ally, not something to be feared. You can simply copy and paste the following commands to compile bitcoin.
+- Admin previlages. If your user is not in the sudoers list then do the following otherwise skip this.
+
+```
+$ su -  # enter root. It will ask you for the root password.
+# apt-get install sudo  # check if sudo is installed.
+# usermod -aG <username> sudo  # add user to the sudoers list
+# reboot  # reboot computer
+```
```
Comment (shannona) [inline A1_0_Compiling_Bitcoin_from_Source.md:31]: I don't think you have to reboot, just log the user out and in again. You should double-check that.

Comment (jodobear) [reply A1_0_Compiling_Bitcoin_from_Source.md:31]: I thought so too, but i double checked again and i had to reboot.

**Comment by jodobear** [comment]
> Also, let me know a proper way to credit this, as I want to make sure we have full credit for anyone who writes a full section or more.

you can credit me as jodobear

File: A1_0_Compiling_Bitcoin_from_Source.md (hunk 1 above)
Comment (shannona) [reply A1_0_Compiling_Bitcoin_from_Source.md:31]: Thanks for checking!

**Pull request merged**